src/app/components/audio-recordings/audio-recording.menus.ts

```diff
@@ -0,0 +1,39 @@
+import { Category, menuLink, menuRoute } from "@interfaces/menusInterfaces";
+import { StrongRoute } from "@interfaces/strongRoute";
+
+export const audioRecordingsRoute = StrongRoute.newRoot().add(
+  "audio_recordings",
+  ({ siteId, regionId, projectId }) => ({ siteId, regionId, projectId })
+);
```

Comment:
I wasn't thinking a shallow route would be needed - i think always nested under project/site/point

you may have done that in the relevant modules though so maybe ignore this comment